content/en/docs/kitex/Getting started/prerequisite.md

```diff
@@ -39,6 +39,12 @@ The code generation tools used in Kitex include the IDL compiler and the kitex t
 
 The IDL compiler can parse IDL and generate corresponding serialization and deserialization code. Kitex supports two types of IDL, Thrift and protobuf, which rely on thriftgo and protoc, respectively, for parsing.
 
+First you may need to set up a domestic proxy:
+
+```bash
+go env -w GOPROXY=https://goproxy.cn
+```
+
```

Comment:
Not needed for English

Reply:
I have deleted the English part, thanks for the suggestion！